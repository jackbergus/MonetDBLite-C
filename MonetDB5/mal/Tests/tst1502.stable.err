stderr of test 'tst1502` in directory 'src/mal` itself:
