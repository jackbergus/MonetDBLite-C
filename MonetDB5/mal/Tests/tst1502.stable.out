stdout of test 'tst1502` in directory 'monetdb5/mal` itself:


function user.main():void;
    x := bat.new(:oid,:bte);
    bat.append(x,1:bte);
    bat.append(x,1:bte);
    bat.append(x,2:bte);
    bat.append(x,2:bte);
    bat.append(x,2:bte);
    bat.append(x,2:bte);
    bat.append(x,3:bte);
    bat.append(x,3:bte);
    y := bat.new(:oid,:dbl);
    le_a := calc.dbl(1);
    bat.append(y,le_a);
    ne_a := calc.dbl(2);
    bat.append(y,ne_a);
    pe_a := calc.dbl(2);
    bat.append(y,pe_a);
    re_a := calc.dbl(2);
    bat.append(y,re_a);
    te_a := calc.dbl(2);
    bat.append(y,te_a);
    ve_a := calc.dbl(3);
    bat.append(y,ve_a);
    xe_a := calc.dbl(1);
    bat.append(y,xe_a);
    af_a := calc.dbl(1);
    bat.append(y,af_a);
    z := bat.new(:oid,:str);
    bat.append(z,"c");
    bat.append(z,"a");
    bat.append(z,"b");
    bat.append(z,"c");
    bat.append(z,"a");
    hf_a := bat.append(z,"a");
    bat.append(z,"c");
    bat.append(z,"a");
    io.print("sort x");
    xs := algebra.subsort(x,false:bit,false:bit);
    io.print(xs);
    io.print("hsort xrev");
    xrev := algebra.subsort(x,true:bit,false:bit);
    io.print(xrev);
end user.main;
[ "sort x" ]
#--------------------------#
# h	t  # name
# void	bte  # type
#--------------------------#
[ 0@0,	1  ]
[ 1@0,	  1	  ]
[ 2@0,	2  ]
[ 3@0,	  2	  ]
[ 4@0,	  2	  ]
[ 5@0,	  2	  ]
[ 6@0,	3  ]
[ 7@0,	  3	  ]
[ "hsort xrev" ]
#--------------------------#
# h	t  # name
# void	bte  # type
#--------------------------#
[ 0@0,	3  ]
[ 1@0,	3  ]
[ 2@0,	2  ]
[ 3@0,	2  ]
[ 4@0,	2  ]
[ 5@0,	2  ]
[ 6@0,	1  ]
[ 7@0,	1  ]


